src/ucp/rndv/rndv_get.c

```diff
@@ -61,8 +61,8 @@ ucp_proto_rndv_get_common_init(const ucp_proto_init_params_t *init_params,
     };
 
     if ((init_params->select_param->dt_class != UCP_DATATYPE_CONTIG) ||
-        !ucp_proto_rndv_op_check(init_params, UCP_OP_ID_RNDV_RECV,
-                                 support_ppln)) {
+        !ucp_proto_rndv_op_rkey_check(init_params, UCP_OP_ID_RNDV_RECV,
```

Comment:
how did it work so far, without this additional check?

Reply:
correct, missed this check https://github.com/openucx/ucx/blob/master/src/ucp/proto/proto_common.c#L504
so new check is needed for rtr_mtype only
updated in place